tools/ports/__init__.py

```diff
@@ -1,4 +1,4 @@
-import sdl, sdl_image, zlib, libpng
-
-ports = [sdl, sdl_image, libpng, zlib]
+import bullet, libpng, ogg, sdl, sdl_image, vorbis, zlib
 
+#If port A depends on port B, then A should be _after_ B
```

Comment:
This is important during linking. If the libraries depend on each other and the order is wrong, we will get some unresolved externals.